docs/styleguide/javascript.rst

```diff
@@ -198,7 +199,11 @@ Example::
 
 
 TODO
-http://javascript.crockford.com/code.html http://jibbering.com/faq/notes/code-guidelines/ http://neil.rashbrook.org/Js.htm
```

Comment:
the last website does not work anymore. found it on archives https://web.archive.org/web/20160614115731/http://neil.rashbrook.org/Js.htm

Reply:
Thanks. yes. I also found it in the archive, but the content is not that important.